tests/playwright-test/reporter-azure.spec.ts

```diff
@@ -0,0 +1,503 @@
+        expect(1).toBe(1);
+      });
+      `
+  }, { reporter: '' });
+  expect(stripAnsi(result.output)).not.toContain('Failed request: (401)');
+  expect(stripAnsi(result.output)).not.toMatch(/azure: (.*)/);
+  expect(result.exitCode).toBe(0);
+});
+
+test('03 testCaseId not specified @azure', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright.config.ts': `
+      module.exports = { 
+        reporter: [
+          ['list'],
+          ['${reporterPath('03')}'],
+          ['azure', { 
+            orgUrl: 'https://dev.azure.com/alex-alex',
+            projectName: 'SampleSample',
+            planId: 4,
+            token: 'token',
+            logging: true,
+          }]
+        ]
+      };
+    `,
+    'a.spec.js': `
+      const { test } = pwt;
+      test('foobar', async () => {
+        expect(1).toBe(1);
+      });
+      `
+  }, { reporter: '' });
+  expect(stripAnsi(result.output)).not.toContain('Failed request: (401)');
+  expect(stripAnsi(result.output)).toContain('azure: Using run 150 to publish test results');
+  expect(stripAnsi(result.output)).toContain('azure: Test foobar - passed');
+  expect(stripAnsi(result.output)).not.toContain('azure: Start publishing: foobar');
+  expect(stripAnsi(result.output)).not.toContain('azure: Result published: foobar');
+  expect(stripAnsi(result.output)).toContain('azure: Run 150 - Completed');
+  expect(result.exitCode).toBe(0);
+});
+
+test('04 incorrect planId @azure', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright.config.ts': `
+      module.exports = { 
+        reporter: [
+          ['list'],
+          ['${reporterPath('04')}'],
+          ['azure', { 
+            orgUrl: 'https://dev.azure.com/alex-alex',
+            projectName: 'SampleSample',
+            planId: 44,
+            token: 'token',
+            logging: true,
+          }]
+        ]
+      };
+    `,
+    'a.spec.js': `
+      const { test } = pwt;
+      test('[3] foobar', async () => {
+        expect(1).toBe(1);
+      });
+      `
+  }, { reporter: '' });
+  expect(stripAnsi(result.output)).not.toContain('Failed request: (401)');
+  expect(stripAnsi(result.output)).toContain('azure: Using run 150 to publish test results');
+  expect(stripAnsi(result.output)).toContain('azure: Test [3] foobar - passed');
+  expect(stripAnsi(result.output)).toContain('azure: Start publishing: [3] foobar');
+  expect(stripAnsi(result.output)).toContain('Could not find test point for test case [3] associated with test plan 44. Check, maybe testPlanId, what you specifiyed, is incorrect');
+  expect(stripAnsi(result.output)).not.toContain('azure: Result published: [3] foobar');
+  expect(stripAnsi(result.output)).toContain('azure: Run 150 - Completed');
+  expect(result.exitCode).toBe(0);
+});
+
+test('05 upload attachments, attachmentsType in not defined - default "screenshot" @azure', async ({ runInlineTest }) => {
```

Comment:
I think we can drop the `@azure` everywhere, since you can run it instead by the file.